app/components/registration-photos/template.hbs

```diff
@@ -0,0 +1,33 @@
+<div class='collapse' id='collapseExample'>
+  <div class='card card-body'>
+    <div>
+      <input
+        aria-label='bird1'
+        type='image'
+        class='register-photo'
+        src='/assets/images/imagesForUserRegistration/owl.jpg'
```

Comment:
Consider to change folder name in kebab-case style:
```suggestion
        src='/assets/images/images-for-user-registration/owl.jpg'
```

Reply:
Unfortunately it doesn't catch photo when I change to kebab style.